src/please.go

```diff
@@ -169,9 +169,10 @@ var opts struct {
 	} `command:"cover" description:"Builds and tests one or more targets, and calculates coverage."`
 
 	Debug struct {
-		Debugger string `short:"d" long:"debugger" description:"Name of supported debugger"`
-		Port     int    `short:"p" long:"port" description:"Debugging server listen port"`
-		Args     struct {
+		Debugger  string `short:"d" long:"debugger" description:"Name of supported debugger"`
+		Port      int    `short:"p" long:"port" description:"Debugging server listen port"`
+		BuildOnly bool   `long:"build_only" description:"Only builds the artifacts skipping the debug session."`
```

Comment:
This is used by vscode to access third party sources included in the pex. If the pex is executed in a sandboxed environment we can't access them.